### PR TITLE
Added Rigetti ParametricQuantumSolver.

### PR DESCRIPTION
**Pull request by RudiP4t1qbit**

### CODE REVIEW

File: openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
+#   Copyright 2019 1QBit
```
Comment (ValentinS4t1qbit) [inline openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py]: 2019-2020 ? Do we need a PR to update all of these? Do we care ? I don t kn ow much about copyright.

File: openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+    Uses the CCSD method to solve the electronic structure problem.
+    PySCF program will be utilized.
+    Users can also provide a function that takes a `pyscf.gto.Mole`
+    as its first argument and `pyscf.scf.RHF` as its second.
+
+    Attributes:
+        optimized_amplitudes (list): The optimized UCCSD amplitudes.
+        of_mole (openfermion.hamiltonian.MolecularData): Molecular Data in Openfermion.
+        f_hamiltonian (openfermion.ops.InteractionOperator): Fermionic Hamiltonian.
+        qubit_hamiltonian (openfermion.transforms.jordan_wigner): Qubit Hamiltonian.
```
Comment (ValentinS4t1qbit) [inline openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py]: I realize here that there are a bunch of different fields in the different backends, and that so far I havent thought about making sure this chunk was updated...

Maybe that can be the object of another PR, doing it for IBM as well.

Comment (RudiP4t1qbit) [reply openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py]: updated, thanks!

File: openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py (hunk 1 above)
Comment (RudiP4t1qbit) [reply openqemist/quantum_solvers/rigetti/rigetti_parametric_solver.py]: I think we can do this later.
